src/lib.rs

```diff
@@ -655,6 +655,14 @@ impl fmt::Display for Mnemonic {
 	}
 }
 
+impl fmt::Debug for Mnemonic {
+	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+		f.debug_struct("Mnemonic")
+			.field("lang", &self.language())
+			.finish_non_exhaustive()
```

Comment:
Ah, unfortunately this is only available since rustc 1.53 (see https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html#method.finish_non_exhaustive). Given that we currently still support 1.41, we should probably just use `finish` for the time being (preferably leaving a TODO comment to switch to `finish_non_exhaustive` once we drop that MSRV requirement.